docs-new/docs/graphql/core/security-disclosure.mdx

```diff
@@ -1,25 +1,24 @@
-<div class="meta" description="Hasura security vulnerability protocol"
-keywords="hasura, docs, security, security disclosure, vulnerability">
-
-</div>
+---
+sidebar_label: Security vulnerability protocol
```

Comment:
This is only required if the heading (`# Security Vulnerability Protocol`) and sidebar label doesn't match. Since they do match here, can be ignored.

Reply:
Ack! Done.